Sources/PointFree/HtmlCssInliner.swift

```diff
@@ -62,10 +62,10 @@ private func applyInlineStyles(element: Element, stylesheet: Stylesheet) -> Elem
     ?? ""
 
   let newStyles = [
-    currentStyles,
-    classStyles,
     elemStyles,
-    idStyles
+    classStyles,
+    idStyles,
+    currentStyles,
```

Comment:
re-ording this to be in line with CSS specificity: tag styles < class styles < id styles < inline styles.